scripts/data_transfer/copy_chr22_skat_input.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gsutil -m mv gs://cpg-tob-wgs-main/CellRegMap_input_files/plink_files/plink_chr22.bed gs://cpg-tob-wgs-test/v0/skat/
```

Comment:
do these need to be moves from main to test, or copies?

Reply:
copied, good catch!!!